### PR TITLE
Update ROADMAP

### PR DESCRIPTION
**Pull request by jdorfman**
* Completed 1.3.0
* Added 1.4.0 (TBA)

### CODE REVIEW

**Comment by jdorfman** [comment]
@flaper87 @tzuryby  need help filling out 1.4.0. This is a CNCF requirement

File: ROADMAP.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
 # Roadmap
 
-## Version: 1.1.0
 
-Timeline: TBD
-Status: In-progress 👩‍💻
+## Version: 1.4.0
+
+Timeline: TBA
```
Comment (flaper87) [inline ROADMAP.md:6]: Still defining 1.4.0 on our end. I'd like to get the RELEASE file in before filling in the data here. That way we have a detailed explanation of what the release schedule looks like.

Comment (jdorfman) [reply ROADMAP.md:6]: Cool thanks

**Comment by flaper87** [comment]
closing in favor of https://github.com/curiefense/curiefense/pull/238 as per https://github.com/curiefense/curiefense/pull/235#issuecomment-798770100

**Pull request closed without merging**